### DISCUSSION TITLE
What can Adonis.Js do?

### DISCUSSION THREAD

**Discussion by CordialGit** [adonisjs/core#4631] [General]
I heard about Adonis.JS today.

There's this talk that it is opinionated. 

While I don't care about Adonis being opinionated, I just want to know if Adonis can solve my problems:

1.) can I build Scalable Apps and any type of API with Adonis?

2.) Can I build Micro-Services with Adonis?

3.) Can we integrate Postgres Database with an Adonis powered backend?

4.) Will it be possible to connect Adonis with a frontend built with Svelte and SvelteKit?

Regards.

**Comment by RomainLanz** [comment]
Maybe take some time to read the documentation and our landing page before asking?
